airflow/www/views.py

```diff
@@ -1247,7 +1247,7 @@ def task(self):
         # Color coding the special attributes that are code
         special_attrs_rendered = {}
         for attr_name in wwwutils.get_attr_renderer():
-            if hasattr(task, attr_name):
+            if getattr(task, attr_name, None) is not None:
```

Comment:
This is to not show attributes that are `None` -- that pollutes the TaskInstance screen.